product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/http/HTTPUtils.java

```diff
@@ -206,4 +206,67 @@ public static JSONObject invokeApiAndGetResponse(String url, String request, Str
         String responsePayload = getResponsePayload(httpResponse);
         return new JSONObject(responsePayload);
     }
+
+    /**
+     * Invoke a pox action and assert
+     *
+     * @param serviceUrl proxy service url
+     * @param request the request to transform
+     * @param contentType content-type of the request
+     * @param messageId message id to be sent a header
+     * @param expectedResponse expected response after transformation
+     * @param statusCode expected status code
+     * @param testcaseName testcase name to be logged during a test failure
+     * @throws IOException
+     * @throws XMLStreamException
+     */
+    public static void invokePoxActionAndAssert(String serviceUrl, String request, String contentType, String messageId,
```

Comment:
Let's rename this function since POX does not have bound action as SOAP. For example we can rename as invokePoxEndpointAndAssert
Use similar name for invokePoxActionAndCheckContains() as well

Reply:
Done.